eng/BootStrapMsBuild.targets

```diff
@@ -35,6 +35,7 @@
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Newtonsoft.Json'))' == 'true'" />
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('NuGetSdkResolver'))' == 'true'" />
         <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Microsoft.Extensions.'))' == 'true'" />
+        <_NuGetRuntimeDependencies Include="%(RuntimeCopyLocalItems.Identity)" Condition="'@(RuntimeCopyLocalItems->Contains('Microsoft.VisualStudio.'))' == 'true'" />
```

Comment:
Is this really not needed? I think we should update the version of this assembly in our updated SDK, because if the SDK is going to take an update it'll flow through us first.

Reply:
It's not needed _for this PR_ since we're not bumping the SolutionPersistence library. But when we do, we should use the new version in the bootstrapped SDK, instead of the old version from the LKG SDK, right?